### PR TITLE
Downcase search name parameter

### PR DESCRIPTION
**Pull request by patrickvinograd**
Fixes https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1694

### CODE REVIEW

File: app/controllers/v0/institutions_controller.rb [hunk 1]
```diff
@@ -41,6 +41,7 @@ def normalized_query_params
       query = params.deep_dup
       query.tap do
         query[:name].try(:strip!)
+        query[:name].try(:downcase!)
```
Comment (saneshark) [inline app/controllers/v0/institutions_controller.rb]: just making sure name is always a string and not an array of strings or hash.

File: spec/controllers/v0/institutions_controller_spec.rb [hunk 2]
```diff
@@ -32,6 +32,13 @@
       expect(response.content_type).to eq('application/json')
       expect(response).to match_response_schema('institutions')
     end
+
+    it 'search returns case-insensitive results' do
+      get :index, name: 'CHICAGO', version: 1
```
Comment (saneshark) [inline spec/controllers/v0/institutions_controller_spec.rb]: Should we maybe improve this to also test 'CHICAGO Illinois' or something like that? Which would return 'University of Illinois Chicago' for example, but not return 'University of Chicago'.

**Review by saneshark — APPROVED** [review]

File: spec/controllers/v0/institutions_controller_spec.rb (hunk 2 above)
Comment (patrickvinograd) [inline spec/controllers/v0/institutions_controller_spec.rb]: Added a test case with mixed case and space in the search term. Passes. Merging.

Comment (saintsoup52) [inline spec/controllers/v0/institutions_controller_spec.rb]: ED actually publishes a great school alias list that catch situations like this. We use the alias list at CFPB and is already in the raw data ingested by the data service.

**Pull request merged**